libraries/AP_InertialSensor/AP_InertialSensor.cpp

```diff
@@ -598,6 +598,21 @@ void AP_InertialSensor::_start_backends()
     }
 }
 
+void AP_InertialSensor::synchronize_fourier_phase(float instant_heading)
+{
+	_backends[0]->synchronize_fourier_phase(instant_heading);
```

Comment:
why force to only use _backends[0]? Do we need to check _backend_count and/or nullptr?

Reply:
if these three functions stay just a single line then I suggest moving them to the header file. See get_last_update_usec() as en example.

Reply:
we could use all the backends, but I wanted to adopt the baby step attitude and not pushing the changes to far.